build.sbt

```diff
@@ -27,13 +27,21 @@ lazy val tests = project
   .settings(moduleName := "magnolia-tests")
   .settings(
     addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full),
+    initialCommands in console := """import magnolia.tests._; import magnolia.examples._;""",
     libraryDependencies ++= Seq(
       "com.fommil" %% "deriving-macro" % "0.9.0",
-      "com.fommil" %% "scalaz-deriving" % "0.9.0"
+      "com.fommil" %% "scalaz-deriving" % "0.9.0",
```

Comment:
Looks like `scalaz-deriving` crept back in... can we drop it?

Reply:
done